backend/tracim_backend/lib/webdav/dav_provider.py

```diff
@@ -38,9 +145,16 @@ def __init__(
         self._app_config = app_config
         self._session = None
         self._plugin_manager = plugin_manager
+        self.processed_path = None
+        self.processed_destpath = None
 
     def set_path(self, path: str):
-        self.path = path
+        self.processed_path = ProcessedWebdavPath(
+            path=path,
+            current_user=self.current_user,
+            session=self.dbsession,
+            app_config=self.app_config,
+        )
 
     @property
     def root_path(self) -> str:
```

Comment:
```suggestion
    def root_path(self) -> typing.Optional[str]:
```

Reply:
are you sure ? `self.environ["http_authenticator.realm"]` may be None ?